guides/common/modules/proc_adding-a-bonded-interface.adoc

```diff
@@ -29,7 +29,7 @@ These can be physical interfaces or VLANs.
 
 * *Bond options*: Specify a space-separated list of configuration options, for example *miimon=100*.
 ifdef::satellite[]
-For more information on configuration options for bonded interfaces, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_and_managing_networking/index#configuring-network-bonding_configuring-and-managing-networking[Configuring network bonding] in the _{RHEL} Configuring and Managing Networking_ guide.
+For more information on configuration options for bonded interfaces, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_networking/index#configuring-network-bonding_configuring-and-managing-networking[Configuring network bonding] in the _{RHEL} Configuring and Managing Networking_ guide.
```

Comment:
```suggestion
For more information on configuration options for bonded interfaces, see {RHELDocsBaseURL}8/html-single/configuring_and_managing_networking/index#configuring-network-bonding_configuring-and-managing-networking[Configuring network bonding] in _{RHEL}{nbsp}8 Configuring and Managing Networking_.
```

Reply:
If there is only one version of the EL link mentioned, I opted out of having `version`, as I observed the procedure remains the same regardless of the EL version.